### ISSUE TITLE
Grandfathered errors from the starter template

### ISSUE THREAD

**Issue by sergei-kalugin** [MADStudioNU/the-pulter-project#542]
![Screen Shot 2021-03-03 at 11 02 23 AM](https://user-images.githubusercontent.com/2558705/109842868-1e9f2800-7c10-11eb-8c6f-a67f3097905f.png)

At some point, we'll need to do some cleanup of TEI that has been copy-pasted countless times at this point. There are typos and formatting issues.

**Comment by LeahKnight** [comment]
@sergei-kalugin Yes, thank you for noticing those! Are "futher" and "miniscule" (visible above) the extent of the problems? If you see more, could you document them? We can certainly tackle this.
